templates/shared/repo_search.tmpl

```diff
@@ -0,0 +1,75 @@
+<div class="ui secondary filter menu gt-ac gt-mx-0">
+	<form class="ui form ignore-dirty gt-f1">
+		<input type="hidden" name="sort" value="{{$.SortType}}">
+		<input type="hidden" name="language" value="{{$.Language}}">
+		<div class="ui fluid action input">
+			{{template "shared/searchinput" dict "Value" .Keyword}}
+			{{if .PageIsExploreRepositories}}
+				<input type="hidden" name="only_show_relevant" value="{{.OnlyShowRelevant}}">
+			{{else if .TabName}}
+				<input type="hidden" name="tab" value="{{.TabName}}">
+			{{end}}
+			<button class="ui primary button">{{ctx.Locale.Tr "explore.search"}}</button>
+		</div>
+	</form>
+	{{$tabQuery := ""}}
+	{{if .TabName}}{{$tabQuery = printf "tab=%s&" (QueryEscape .TabName)}}{{end}}
+	{{$languageQuery := ""}}
+	{{if .TabName}}{{$languageQuery = printf "language=%s&" (QueryEscape .Language)}}{{end}}
+	{{$queryParams := printf "%s%sq=%s" $tabQuery $languageQuery (QueryEscape .Keyword)}}
```

Comment:
It doesn't seem right. Suppose `tabQuery="tab=a"` and `languageQuery="language=b"`, then here it gets `tab=alanguage=b` ?